crates/ty/src/lib.rs

```diff
@@ -296,6 +296,11 @@ impl MainLoop {
                             tracing::warn!("No python files found under the given path(s)");
                         }
 
+                        // TODO: We should have an official flag to silence workspace diagnostics.
```

Comment:
I, at first, didn't understand what features is needed to replace this environment variable usage but I think I understand now. What we need is a `--quiet` or similar flag. Would you mind opening an issue in the ty repository for adding a flag to suppress diagnostic printing (and mention Ruff's `--quiet` and `--silent` flags)

Reply:
Created https://github.com/astral-sh/ty/issues/772